src/components/map-controls.js

```diff
@@ -240,7 +240,7 @@ export default class MapControls extends PureComponent {
   }
 
   _unproject(pos) {
-    const viewport = new PerspectiveMercatorViewport(Object.assign({}, this.props));
+    const viewport = new PerspectiveMercatorViewport(this.props);
     const lngLat = this.props.unproject ?
```

Comment:
I think we can remove this.props.unproject now (it was there to allow injecting mapbox version during development)